src/t8_cmesh/t8_cmesh_commit.cxx

```diff
@@ -90,9 +90,9 @@ t8_cmesh_add_attributes (const t8_cmesh_t cmesh, sc_hash_t *ghost_ids)
   const t8_stash_t stash = cmesh->stash;
   t8_locidx_t ltree;
   size_t si, sj;
-  t8_ghost_facejoin_t *temp_facejoin, **facejoin_pp; /* used to lookup global ghost ids in the hash */
+  t8_ghost_facejoin_t **facejoin_pp; /* used to lookup global ghost ids in the hash */
 
-  temp_facejoin = T8_ALLOC_ZERO (t8_ghost_facejoin_t, 1);
+  t8_ghost_facejoin_t temp_facejoin = { 0, 0, 0 };
```

Comment:
The alloc before only allocated memory for one value

Reply:
This is still a single value, but all members of the struct are initialized with 0 to be on par with T8_ALLOC_ZERO.